docs/sources/articles/https.md

```diff
@@ -96,6 +96,24 @@ Now sign the key:
     Getting CA Private Key
     Enter pass phrase for ca-key.pem:
 
+After generating `cert.pem` and `server-cert.pem` you can safely remove the
+two certificate signing requests:
+
+    $ rm -v client.csr server.csr
+
+With a default `umask` of 022 your secret keys will be *world-readable* and
+writable for you and your group.
+
+To remove write permissions for your keys in order to protect them from accidental
+damage and make them only readable to you issue the following file mode changes:
```

Comment:
Need to break up this run-on. Suggest something like "In order to protect your keys from accidental damage, you will want to remove their write permissions. To make them read-only, change file modes as follows: